<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>659</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Каталог книг</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="profit">
      <property name="styleSheet">
       <string notr="true">font-family: Comic Sans MS;
  font-size: 15px;

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>общая прибыль: 999</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font-family: Comic Sans MS;
  font-size: 15px;

</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Каталог книг &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentBook">
            <property name="styleSheet">
             <string notr="true">font-family: Comic Sans MS;
  font-size: 15px;

</string>
            </property>
            <property name="text">
             <string>Пока что вы не взаимодействовали ни с какой из книг</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="poiskBut">
              <property name="styleSheet">
               <string notr="true">font-family: Comic Sans MS;
  font-size: 15px;

</string>
              </property>
              <property name="text">
               <string>Поиск</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="poiskEdit"/>
            </item>
            <item>
             <widget class="QComboBox" name="parametrs">
              <property name="styleSheet">
               <string notr="true">font-family: Comic Sans MS;
  font-size: 15px;

</string>
              </property>
              <item>
               <property name="text">
                <string>Выберите параметр поиска</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Название</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Автор</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Цена (Вводить в формате' x - y'), где x - минимальная цена, а y - максимальная</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="BooksList">
        <property name="styleSheet">
         <string notr="true">font-family: Comic Sans MS;
  font-size: 15px;

</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="updateBut">
            <property name="styleSheet">
             <string notr="true">font-family: Comic Sans MS;
  font-size: 15px;

</string>
            </property>
            <property name="text">
             <string>Обновить каталог</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="editBut">
            <property name="styleSheet">
             <string notr="true">font-family: Comic Sans MS;
  font-size: 15px;

</string>
            </property>
            <property name="text">
             <string>Редактировать книгу</string>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="textBut">
            <property name="styleSheet">
             <string notr="true">font-family: Comic Sans MS;
  font-size: 15px;

</string>
            </property>
            <property name="text">
             <string>Текстовый формат</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="delBut">
            <property name="styleSheet">
             <string notr="true">font-family: Comic Sans MS;
  font-size: 15px;

</string>
            </property>
            <property name="text">
             <string>Удалить книгу</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addBut">
            <property name="styleSheet">
             <string notr="true">
background-color: #BA8A8A;
font-family: Comic Sans MS</string>
            </property>
            <property name="text">
             <string>Добавить новую книгу</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../pictures/newbut.png</normaloff>../pictures/newbut.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addAuthorBut">
            <property name="styleSheet">
             <string notr="true">background-color: #BA8A8A;
font-family: Comic Sans MS</string>
            </property>
            <property name="text">
             <string>Добавить/удалить автора</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../pictures/newauthor.png</normaloff>../pictures/newauthor.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
